docs/en/integrations/language-clients/java/index.md

```diff
@@ -139,6 +139,55 @@ CompletableFuture<List<ClickHouseResponseSummary>> future = ClickHouseClient.sen
 List<ClickHouseResponseSummary> results = future.get();
 ```
 
+#### Named Parameters
+
+As being shown in the previous examples, you can pass parameters by name rather than relying solely on their position in the parameter list.
+This capability is available using `params` function.
```

Comment:
```suggestion
You can pass parameters by name rather than relying solely on their position in the parameter list. This capability is available using `params` function.
```